netmanager/src/views/components/DataDisplay/DeviceManagement/ManagementStats.js

```diff
@@ -202,18 +176,12 @@ export default function ManagementStat() {
           />
 
           <ChartContainer
-            title={"leaderboard"}
+            title={'Leaderboard in the last 24hours'}
             controller={
```

Comment:
Changed Leaderboard title

Reply:
Thanks, in the near future, we could add a space between 24 and hours.

Also, please plan to offer more toggle options for the users when it comes to this board since the endpoint already supports it, more like:

1. last 24 hours
2. Last 3 days
3. Last 4 days
4. Last 5 days.

Relatedly, @123MwanjeMike , this is the endpoint to use in Grafana for the uptime visualisations in case you were still looking for it. Please get in touch with @Codebmk to learn more about these analytics endpoints and how we can take advantage of them in the Grafana visualisation tools.